src/cli/commands/compile.rs

```diff
@@ -0,0 +1,29 @@
+use std::{fs::File, io::Write, path::PathBuf, process::Command};
+
+const JSON_FILE_EXTENTION: &str = "json";
+
+pub fn compile(program: &PathBuf) -> PathBuf {
+	// Use cairo-compile binary in order to compile the .cairo file
+	let compiled_output = Command::new("cairo-compile")
+		.args([&program])
+		.output()
+		.expect("cairo-compile command failed to start");
+
+	// Retrieve only the file name to create a clean compiled file name.
+	let filename = program.file_stem().unwrap();
```

Comment:
Good practice: each time you use an unwrap, add a comment explaining why it is safe to unwrap there.
Here it's because the call to `cairo-compile` would have failed if `program` was not terminated by `.cairo`